src/server/zset_family.cc

```diff
@@ -2393,6 +2393,21 @@ OpResult<MScoreResponse> ZSetFamily::ZGetMembers(CmdArgList args, ConnectionCont
   return cntx->transaction->ScheduleSingleHopT(std::move(cb));
 }
 
+double ZSetFamily::ExtractUnitOrReply(std::string_view arg, ConnectionContext* cntx) {
```

Comment:
it does not have to be ZSetFamily member, just add an auxillary function in anonymous namespace in this file.